
CondaError: Run 'conda init' before 'conda activate'

cd /data/yangdi/XinHaiLLM-fork/frontend && npm run serve
cd /data/yangdi/XinHaiLLM-fork/backend/src && python -m xinhai.controller --host 0.0.0.0 --port 5000
cd /data/yangdi/XinHaiLLM-fork/backend/src && CUDA_VISIBLE_DEVICES=0 LD_LIBRARY_PATH=/home/tanminghuan/anaconda3/lib/ CONTROLLER_ADDRESS=http://localhost:5000 MODEL_NAME=paddleocr WORKER_ADDRESS=http://localhost:40000 WORKER_HOST=0.0.0.0 WORKER_PORT=40000 python -m xinhai.workers.ocr
cd /data/yangdi/XinHaiLLM-fork/backend/src && CUDA_VISIBLE_DEVICES=0 PYTHONPATH=/data/yangdi/XinHaiLLM-fork/related_repos/LLaMA-Factory/src CONTROLLER_ADDRESS=http://localhost:5000 MODEL_NAME=Qwen1.5-7B-Chat WORKER_ADDRESS=http://localhost:40001 WORKER_HOST=0.0.0.0 WORKER_PORT=40001 python -m xinhai.workers.llm --model_name_or_path /data2/public/pretrained_models/Qwen1.5-7B-Chat --template qwen --infer_backend vllm --vllm_enforce_eager --infer_dtype float16
cd /data/yangdi/XinHaiLLM-fork/backend/src && CUDA_VISIBLE_DEVICES=0 CONTROLLER_ADDRESS=http://localhost:5000 MODEL_NAME=knowledge WORKER_ADDRESS=http://localhost:40002 WORKER_HOST=0.0.0.0 WORKER_PORT=40002 PRO_KNOWLEDGE_DB_PATH=/data/pretrained_models/ProDB-bge-1.5-300 SS_KNOWLEDGE_DB_PATH=/data/pretrained_models/KnowDB-bge-1.5-300 EMBEDDING_MODEL_PATH=/data/pretrained_models/bge-large-zh-v1.5 RERANKER_MODEL_PATH=/data/pretrained_models/maidalun/bce-reranker-base_v1 python -m xinhai.workers.knowledge
cd /data/yangdi/XinHaiLLM-fork/backend/src && CUDA_VISIBLE_DEVICES=0 CONTROLLER_ADDRESS=http://localhost:5000 MODEL_NAME=storage WORKER_ADDRESS=http://localhost:40003 WORKER_HOST=0.0.0.0 WORKER_PORT=40003 DB_PATH=/data/pretrained_models/CBT-DB EMBEDDING_MODEL_PATH=/data/pretrained_models/bge-large-zh-v1.5 python -m xinhai.workers.storage
